integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInImage.java

```diff
@@ -215,7 +215,7 @@ public void testIstioDomainHomeInImage() {
     assertTrue(checkConsole, "Failed to access WebLogic console");
     logger.info("WebLogic console is accessible");
 
-    Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
+    Path archivePath = Paths.get(ITTESTS_DIR, "../operator/integration-tests/apps/testwebapp.war");
```

Comment:
should testwebapp.war be moved to under integration-tests instead of under operator?

Reply:
Probably... I was assuming that there was some reason that these files were already in this odd directory. I don't know a lot about these integration-tests. They are separate from the tests in the integration-tests module and bash-based.

Reply:
I'm going to leave these alone for now and open a follow-on tracking issue. We're not supposed to have checked-in jars, ears, or wars. I see that other code in the main integration-tests module are building fairly similar applications. These pre-built applications need to be deleted and replaced with references to an application that we build.